site/en/blog/renderingng/index.md

```diff
@@ -0,0 +1,554 @@
+GPU acceleration provides an enormous speedup for most content,
+because every pixel can be processed in parallel.
+It is also an effective method for improving performance on low-end devices, which tend to still have a GPU.
+It began in 2014 and completed in 2020.
+
+<table>
+  <thead>
+    <tr>
+      <th>Year</th>
+      <th>Progress</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2014</td>
+      <td>Canvas support. Shipped on opt-in content on Android.</td>
+    </tr>
+    <tr>
+      <td>2016</td>
+      <td>Ship on Mac.</td>
+    </tr>
+    <tr>
+      <td>2017</td>
+      <td>GPU is used on &gt;60% of Android page views.</td>
+    </tr>
+    <tr>
+      <td>2018</td>
+      <td>Ship on Windows, ChromeOS, and Android Go.</td>
+    </tr>
+    <tr>
+      <td>2019</td>
+      <td>Threaded GPU Rasterization.</td>
+    </tr>
+    <tr>
+      <td>2020</td>
+      <td>Ship remaining Android content.</td>
+    </tr>
+    <tr>
+  </tbody>
+</table>
+
+### Threaded scrolling, animations, and decode
+
+A long-term effort to move all scrolling, non-layout-inducing animations,
+and image decoding off of the main thread. It began in 2011 and is ongoing.
+
+<table>
+  <thead>
+    <tr>
+      <th>Year</th>
+      <th>Progress</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2011</td>
+      <td>Initial support for threaded scroll and animation.</td>
+    </tr>
+    <tr>
+      <td>2015</td>
+      <td>Layer squashing.</td>
+    </tr>
+    <tr>
+      <td>2016</td>
+      <td>Universal overflow scrolling.</td>
+    </tr>
+    <tr>
+      <td>2017</td>
+      <td>Image decodes on compositor thread.</td>
+    </tr>
+    <tr>
+      <td>2018</td>
+      <td>Image Animations on compositor thread.</td>
+    </tr>
+    <tr>
+      <td>2020</td>
+      <td>Always composite fixed-position.</td>
+    </tr>
+    <tr>
+      <td>2021</td>
+      <td>Percentage transform animations, SVG animations.</td>
+    </tr>
+    <tr>
+  </tbody>
+</table>
+
+### Viz
+
+A centralized raster and draw process for Chromium that increases throughput,
+optimizes memory, and allows optimal use of hardware capabilities.
+It also has other benefits less visible to web developers but very visible to users,
+such as unblocking Site Isolation and decoupling the rendering pipeline from browser UI rendering. It began in 2016 and will complete in 2021.
+
+<table>
+  <thead>
+    <tr>
+      <th>Year</th>
+      <th>Progress</th>
+  </thead>
+  <tbody>
+    <tr>
+      <td>2018</td>
+      <td>OOP-R shipped on Android, Mac and Windows.</td>
+    </tr>
+    <tr>
+      <td>2019</td>
+      <td>OOP-D shipped. OOP-R shipped everywhere (except Canvas). SkiaRenderer shipped on Linux.</td>
+    </tr>
+    <tr>
+      <td>2020</td>
+      <td>SkiaRenderer shipped on Windows & Android. Vulkan shipped on Android.</td>
+    </tr>
+    <tr>
+      <td>2021</td>
+      <td>SkiaRenderer shipped on Mac (and ChromeOS soon).</td>
+    </tr>
+    <tr>
+  </tbody>
+</table>
+
```

Comment:
```suggestion

Definitions of terms in the chart above:

OOP-D
: Out of process display compositor.
Display compositing is the same kind of activity as an
[OS compositor](https://en.wikipedia.org/wiki/Compositing_window_manager).
Out of process means doing it in the Viz process instead of the web page's render process or the browser UI process.

OOP-R
: Out of process raster. Raster is converting display lists into pixels.
Out of process means doing it in the Viz process instead of the web page's render process.

SkiaRenderer
: A new display compositor implementation that can support execution on a range of different
underlying GPU APIs such as Vulkan, D3D12 or Metal.

```